Error in deploying full-text-search function

The module referenced in [full-text-seach's index.js](https://github.com/firebase/snippets-node/blob/5a274ccf712d4c642a0afb424b3908649ed2f768/firestore/full-text-search/functions/index.js#L2 ) file does not exist.

On deploying, it gives the following error:
```
Error: Error parsing triggers: Cannot find module 'firebase-functions/lib/providers/datastore'
```

which is probably due to datastore being renamed to firestore?

--- a/firestore/full-text-search/functions/index.js
+++ b/firestore/full-text-search/functions/index.js
@@ -1,5 +1,5 @@
 const functions = require("firebase-functions");
-const firestore = require("firebase-functions/lib/providers/datastore");
+const firestore = require("firebase-functions/lib/providers/firestore");
 const Firestore = require("@google-cloud/firestore");
 
 const algoliasearch = require("algoliasearch");

